sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3828,6 +3828,15 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val LEGACY_ARRAY_SORT_HANDLES_COMPARISON_RESULT_NULL_VALUE_AS_ZERO =
+    buildConf("spark.sql.legacy.arraySortHandlesComparisonResultNullAsZero")
```

Comment:
qq should it be covered by ansi config @gengliangwang ?

Reply:
I don't think so, legacy config is for legacy behaviors.

Reply:
+1 with @cloud-fan